releases/unreleased/fix-repo-when-is-a-submodule.yml

```
@@ -0,0 +1,8 @@
+---
+title: Fix repo when is a submodule
+category: fixed
+author: Jose Javier Merchante <jjmerchante@bitergia.com>
+issue: null
+notes: >
+  A variable was referenced before assignment
+  when the repository was a submodule.
```

The explanation should be other. What type of error we fix not what caused it. Remember that this is for users of the tools, not developers. What do you think?

```suggestion
notes: >
  Release tools didn't work when the project is under a git submodule.
```